debezium-core/src/main/java/io/debezium/config/ValidationResults.java

```diff
@@ -14,11 +14,11 @@
 
 public class ValidationResults {
```

Comment:
We really should be writing javadocs. Self descriptive code is a myth and in this case I find it a bit hard to get an exact understanting of how the code works / is intended to be used.

Reply:
Self descriptive code is not myth but public classes/interfaces esp. those that represent a contract should have at least class levele JavaDoc.

Reply:
A very good point on that "helper" interface which is solely used to pass an anonymous method / lambda. That could really require some description here! Will add it.